static/js/public/snap-details/videos.js

```diff
@@ -1,29 +1,71 @@
-function youtube(holderEl, url) {
-  const frame = document.createElement("iframe");
-  frame.id = "ytplayer";
-  frame.type = "text/html";
-  frame.width = 440;
-  frame.height = 248;
-  frame.src = `${url}?autoplay=0&origin=${window.location.href}`;
-  frame.setAttribute("frameborder", "0");
-  holderEl.appendChild(frame);
+function vimeo() {
+  const vimeoPlayerScript = document.createElement("script");
+  vimeoPlayerScript.src = "https://player.vimeo.com/api/player.js";
```

Comment:
As vimeo doesn't support "?muted=1" as a query param (anymore) we need to load the iframe then manipulate the volume to 0 with JS.

I don't add "?autoplay=1" to vimeo so that it starts paused until the volume is set to 0.